README.md

```diff
@@ -43,8 +43,10 @@ pip install -e .
 # If using macOS High Sierra or higher, run this before run setup, to allow multithreading
 # export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
 
-# Download required data files.
-nlg-eval --setup
+# Download required data and external code files.
+# If you don't like the default (~/.cache/nlgeval), specify a path instead of "default"
```

Comment:
Can we make it clearer what this is? E.g. "If you don't like the default executable path ..." ? If that's even what it is.
Also many programs just make their own folder under `~/`, e.g. `~/.nlgeval` is probably fine for all of our stuff.

Reply:
Ah I see that xdg is encouraging this. I still think it's uncommon.

Reply:
I'll add the clarification you suggested. It's rather common actually, my .cache and .config directories have a bunch of stuff in it.